rocketmq-connect/rocketmq-connect-runtime/src/main/java/org/apache/rocketmq/connect/runtime/connectorwrapper/Worker.java

```diff
@@ -257,10 +257,10 @@ public synchronized void startTasks(Map<String, List<ConnectKeyValue>> taskConfi
 
                 if (task instanceof SourceTask) {
                     DefaultMQProducer producer = new DefaultMQProducer();
-                    producer.setNamesrvAddr(keyValue.getString(RuntimeConfigDefine.NAMESRV_ADDR));
+                    producer.setNamesrvAddr(connectConfig.getNamesrvAddr());
                     String rmqProducerGroup = keyValue.getString(RuntimeConfigDefine.RMQ_PRODUCER_GROUP);
                     if (StringUtils.isEmpty(rmqProducerGroup)) {
-                        rmqProducerGroup = connectConfig.getRmqProducerGroup();
+                        rmqProducerGroup = new StringBuilder().append(connectConfig.getRmqProducerGroup()).append(System.currentTimeMillis()).toString();
```

Comment:
It is recommended to reuse the same producer as much as possible to avoid creating many remote channels.

Reply:
sink not use, so i not fix it .If need to repair him urgently ，i will do it.

Reply:
> sink does not used, so i do not fix it .If need to repair him urgently ，i will do it.

Just do it :)